corehq/apps/export/views.py

```diff
@@ -295,113 +407,799 @@ def create_basic_form_checkpoint(index):
     return checkpoint
 
 
-class CreateFormExportView(BaseProjectDataView):
-    urlname = 'create_form_export'
-    page_title = ugettext_noop("Create Form Export")
-    template_name = 'export/create_form_export.html'
+class BaseDownloadExportView(ExportsPermissionsMixin, JSONResponseMixin, BaseProjectDataView):
+    template_name = 'export/download_export.html'
+    http_method_names = ['get', 'post']
+    show_sync_to_dropbox = False  # remove when DBox issue is resolved.
```

Comment:
i think maybe sync to dropbox isn't supported for exports for reasons I don't rememver